libs/use-viewer/src/index.ts

```diff
@@ -223,14 +223,14 @@ const useViewer = ({ handleError, streamAccountId, streamName, subscriberToken }
 
     if (remoteTrackSource) {
       try {
-        await viewer.project(sourceId, remoteTrackSource.projectMapping);
+        await viewer.project(sourceId ?? undefined, remoteTrackSource.projectMapping);
```

Comment:
if it's undefined, it's undefined, do we still need this?

Reply:
This is to stop TypeScript from complaining, it expects `string | undefined` so I'm changing `null` to `undefined` here.